compiler/il/symbol/OMRResolvedMethodSymbol.cpp

```diff
@@ -1821,14 +1821,23 @@ OMR::ResolvedMethodSymbol::insertStoresForDeadStackSlotsBeforeInducingOSR(TR::Co
    TR::TreeTop *prev = induceOSRTree->getPrevTreeTop();
    TR::TreeTop *next = induceOSRTree;
 
-   TR::OSRTransitionTarget pointType = TR::preAndPostExecutionOSR;
+   /*
+    * Any arguments used by this bytecode index should be
+    * kept live
+    */
+   TR_BitVector *deadSymRefs = osrMethodData->getLiveRangeInfo(byteCodeIndex);
```

Comment:
This MR was missing a change so that pre OSR would use these stashed arguments when they were available, rather than copying a reference node. With this added, the copied symrefs must be kept live for the transition, so this is done unconditionally.